themes/hugo-creative-theme/layouts/partials/contact.html

```diff
@@ -0,0 +1,45 @@
+{{ "<!-- CONTACT -->" | safeHTML }}
+<section id="contact">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2 text-center">
+                {{ with .Site.Params.contact.headline }}
+                    <h2 class="section-heading">{{ . }}</h2>
+                {{ end }}
+                {{ if and .Site.Params.contact.headline .Site.Params.contact.description }}
+                    <hr class="primary">
+                {{ end }}
+                {{ with .Site.Params.contact.description }}
+                    <p>{{ . | markdownify }}</p>
+                {{ end }}
+            </div>
+		</div>
+		<br>
+        <div class="row">
+            {{ if .Site.Params.contact.github }}
+                <div class="col-md-3 text-center">
+                    <i class="fab fa-github fa-3x wow bounceIn"></i>
+                    <p><a href="{{ .Site.Params.contact.github }}">GitHub</a></p>
+                </div>
+            {{ end }}
+            {{ if .Site.Params.contact.documentation}}
```

Comment:
there should be a space after `if .Site.Params.contact.documentation` and before `}}` here for consistency.

Reply:
Fixed